dubbo-remoting/dubbo-remoting-netty4/src/main/java/org/apache/dubbo/remoting/transport/netty4/NettyConnectionClient.java

```diff
@@ -374,7 +385,7 @@ public void operationComplete(ChannelFuture future) {
                                     "Failed to connect to server: " + getConnectAddress());
                         }
                     },
-                    1L,
+                    1,
```

Comment:
Please sync with `org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient#calculateReconnectDuration` to get the duration

Reply:
@AlbumenJ 
Thank you for your review.
I have modified the code to get the idle time before reconnection. By default, it will now wait for 60000ms.